txgithub/api.py

```diff
@@ -338,3 +345,94 @@ def create(self, repo_user, repo_name, issue_number, body):
              'issues', issue_number, 'comments'],
             method='POST',
             post=dict(body=body))
+
+
+class ReviewCommentsEndpoint(BaseEndpoint):
+    def getRepoComments(self, repo_user, repo_name):
+        """
+        GET /repos/:owner/:repo/pulls/comments
+        """
+        return self.api.makeRequestAllPages(
+            ['repos', repo_user, repo_name, 'pulls', 'comments'])
+
+    def getPullRequestComments(self, repo_user, repo_name, pull_number):
+        """
+        GET /repos/:owner/:repo/pulls/:number/comments
+
+        :param pull_number: The pull request's number.
+        """
+        return self.api.makeRequestAllPages(
+            ['repos', repo_user, repo_name,
+             'pulls', str(pull_number), 'comments'])
+
+    def getComment(self, repo_user, repo_name, comment_id):
+        """
+        GET /repos/:owner/:repo/pull/comments/:number
+
+        :param comment_id: The review comment's ID.
+        """
+        return self.api.makeRequest(
+            ['repos', repo_user, repo_name,
+             'pulls', 'comments', str(comment_id)])
+
+    def createComment(self, repo_user, repo_name, pull_number,
+                      body, commit_id, path, position):
+        """
+        POST /repos/:owner/:repo/pulls/:number/comments
+
+        :param pull_number: The pull request's ID.
+        :param body: The text of the comment.
+        :param commit_id: The SHA of the commit to comment on.
+        :param path: The relative path of the file to comment on.
+        :param position: The line index in the diff to comment on.
+        """
+        return self.api.makeRequest(
+            ["repos", repo_user, repo_name,
+             "pulls", str(pull_number), "comments"],
+            method="POST",
+            data={"body": body,
```

Comment:
Drive by consistency nit: seems like the rest of the file uses the dict(body=body....) syntax, maybe do the same for consistency. (Ditto for following)